lib/opal/path_reader.rb

```diff
@@ -3,7 +3,7 @@
 
 module Opal
   class PathReader
-    RELATIVE_PATH_REGEXP = %r{#{Opal::REGEXP_START}\.?\.#{File::SEPARATOR}}
+    RELATIVE_PATH_REGEXP = %r{#{Opal::REGEXP_START}\.?\.#{Regexp.quote File::SEPARATOR}}
```

Comment:
Another way to do it is to use `File::SEPARATOR.gsub('\\', '\\\\\\\\')`.
But `Regexp.quote` escapes any characters that would have special meaning in a regular expression (general purpose) + this method is already used in other places in the code.